website/content/docs/platform/k8s/vso/openshift.mdx

```diff
@@ -7,14 +7,23 @@ description: >-
 
 # Run the Vault Secrets Operator on OpenShift
 
-The Vault Secrets Operator may be installed on OpenShift clusters via the embedded OperatorHub or the Helm chart.
+The Vault Secrets Operator may be installed on OpenShift clusters via the embedded OperatorHub or the Helm chart. OpenShift versions 4.12 and later are supported.
 
 ## OperatorHub
 
 The Vault Secrets Operator is certified by Red Hat and therefore included in the [OperatorHub section](https://access.redhat.com/documentation/en-us/openshift_container_platform/4.2/html/operators/olm-understanding-operatorhub) of an OpenShift cluster's web console.
 
 Navigate to the OperatorHub page of your OpenShift cluster and search for `Vault Secrets Operator`, then follow the instructions to install.
 
+### Options
+
+Set the following environment variables [on the subscription](https://access.redhat.com/solutions/5761251) to customize the operator's behavior.
```

Comment:
```suggestion
Set the following environment variables [on the subscription](https://access.redhat.com/solutions/5761251) to customize the Vault behavior.
```
Vault variables don't customize a person's behavior ;)

Reply:
This is referring to a [Kubernetes Operator](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/) (Vault Secrets Operator), not a person who is operating Vault.

Reply:
@schavis can we remove this request for change? (see previous comment)

Reply:
changed the suggestion to clarify the language